### ISSUE TITLE
Nodemcu isn't detected (COM)

### ISSUE THREAD

**Issue by n00bhax** [nodemcu/nodemcu-flasher#29]
the program doesn't see the com and in device manager it's the same. Nothing is connected. I do have an arduino nano which works perfectly and is detected by device manager. I tried 2 different cables and pressing the flash and reset buttons on the esp8266.

Thanks for helping!

**Comment by celgara** [comment]
So most likely the reason for this is a driver issue the driver is cp2102 driver esp8266
 or here is a link hope this helps
 https://www.silabs.com/products/development-tools/software/usb-to-uart-bridge-vcp-drivers 

just unzip the driver and run the exe and you should start seeing the esp8266

**Comment by n00bhax** [comment]
I don't think this will work. The nodemcu was just broken. I bought another one and it worked fine

**Comment by n00bhax** [comment]
thanks for the answer though

**Comment by celgara** [comment]
no problem

**Comment by mlinaje** [comment]
I guess issues should be close since it is not a nodemcu problem, but a USB-to-Serial IC problem.
Look at https://github.com/nodemcu/nodemcu-flasher/issues/31
Close issue 29 and issue 31 :-) 👍

**Comment by tristanlirette** [comment]
> 
> 
> So most likely the reason for this is a driver issue the driver is cp2102 driver esp8266
> or here is a link hope this helps
> https://www.silabs.com/products/development-tools/software/usb-to-uart-bridge-vcp-drivers
> 
> just unzip the driver and run the exe and you should start seeing the esp8266

Worked for me, the COM port was available after installing the driver.
